### PR TITLE
Proxy SSH

### PR DESCRIPTION
**Pull request by jlmorris3827**
Update to the forked ssh to optionally enable proxy SSH servers.   This is for the CRM-GW functionality for TELUS and others.  There will be an infra change afterwards to enable this.

### CODE REVIEW

**Review by ashxjain — APPROVED** [review]
LGTM. Thanks!

File: client.go [hunk 1]
```diff
@@ -177,6 +180,8 @@ func NewNativeClient(user, host, clientVersion string, port int, auth *Auth, hos
 		Config:        config,
 		Hostname:      host,
 		Port:          port,
+		ProxyHost:     proxyHost,
+		ProxyPort:     proxyPort,
```
Comment (gainsley) [inline client.go:177]: Would it make sense to also have a ProxyConfig here in case the proxy config and the final hop config need to be different?

**Review by gainsley — APPROVED** [review]
Looks good! That's pretty cool that you can do ssh proxying just by dialing again.

**Comment by gainsley** [comment]
Btw when integrating with infra, you'll need to create a new release version on this repo 0.0.4 and then set that version in go.mod on edge-cloud and edge-cloud-infra.

**Comment by jlmorris3827** [comment]
Thanks for the review & comments!  I will separate the config for proxy as suggested

**Comment by gainsley** [comment]
Looks good, thanks!

**Pull request merged**